python/datasources/cdc_vaccination_national.py

```diff
@@ -28,14 +28,14 @@
 }
 
 CDC_AGE_GROUPS_TO_STANDARD = {
+    'Ages_<2yrs': '<2',
```

Comment:
Did we talk about options for under 2? In incarceration we use `0-17` for under 18, but I'm open to other ideas like `Under 2` or `<2` we should discuss what's the easiest to understand

Reply:
Great point!

Reply:
Maybe we just do the `0-2` in order to keep it consistent with the rest of the site. What do you think @joshzarrabi?

Reply:
Yeah lets do `0-2`